source/tool/chipsec/hal/acpi_tables.py

```diff
@@ -584,3 +584,27 @@ def __str__( self ):
 ACPI Table Entries:
 %s
 """ %( ''.join( ['0x%016X\n' % addr for addr in self.Entries]))
+
+########################################################################################################
+#
+# FACP Table

```

Comment:
Although the signature is 'FACP', the table name is FADT. Please rename the class to FADT.